### PR TITLE
Add Final Kit Packing day email

### PR DESCRIPTION
**Pull request by Adimote**

### CODE REVIEW

File: SR2019/2018-10-30.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+to: All Volunteers
+subject: We need help packing kits.
+---
+
+Hi Everyone!
+
+This weekend is the last weekend before our kick-off event (Kickstart) and we've got a lot of kits left to pack!
```
Comment (PeterJCLaw) [inline SR2019/2018-10-30.md]: Maybe frame this so it doesn't sound like some are already packed?

File: SR2019/2018-10-30.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+If you're able to make it to the University of Southampton this weekend it would be great if you could help with kit packing and event preparation!
+
+[Click here to help!][volunteer-form]
+
+We'll be working in room 4025 (top floor lecture theatre) of [Building 53][building-53].
```
Comment (PeterJCLaw) [inline SR2019/2018-10-30.md]: Are we confirmed that this is the venue? (Not sure if I missed an update about this)

Comment (PeterJCLaw) [reply SR2019/2018-10-30.md]: Oh, just spotted the following paragraph. Perhaps make this one sound less sure? (and maybe run the two together)

```suggestion
We're anticipating that we'll be working in room 4025 (top floor lecture theatre) of [Building 53][building-53].
```

Comment (Adimote) [reply SR2019/2018-10-30.md]: I'm happy to wager that it'll be Mountbatten, blocking off a building is a rare occurrence.

Comment (Adimote) [reply SR2019/2018-10-30.md]: They probably should have been run together, but I clicked 'send' in the minute between you mentioning the venue and you noticing the following paragraph (sorry!)

**Comment by Adimote** [comment]
Sent! https://us12.admin.mailchimp.com/campaigns/show?id=1530117

**Pull request merged**